addons/mail/static/src/models/rtc_call_viewer/rtc_call_viewer.js

```diff
@@ -120,7 +118,9 @@ function factory(dependencies) {
                 } else if (document.webkitCancelFullScreen) {
                     await document.webkitCancelFullScreen();
                 }
-                this.update({ isFullScreen: false });
+                if (this.exists()) {
+                    this.update({ isFullScreen: false });
+                }
```

Comment:
I wonder what impact it would have to write the field to false before the await?

Reply:
Shouldn't this value be written outside the main condition? If the method is called and no full screen element is found the field should be false probably.